addons/project/models/project.py

```diff
@@ -1601,6 +1601,13 @@ def _ensure_portal_user_can_write(self, fields):
             if field not in self.SELF_WRITABLE_FIELDS:
                 raise AccessError(_('You have not write access of %s field.') % field)
 
+    def _load_records_create(self, values):
```

Comment:
We could rename it into ´vals_list´ like we have in create method. 😇

Reply:
> We could rename it into ´vals_list´ like we have in create method. 😇

@xavierbol Done. FYI If the recurring feature is enabled then a recurring task will be created.